files/20_linux_tboot

```diff
@@ -0,0 +1,246 @@
+#! /bin/sh
```

Comment:
We need to get this replaced with a patch and exec so that we don't GPL up our Apache2 module